packages/wdio-crossbrowsertesting-service/src/launcher.ts

```diff
@@ -6,22 +6,26 @@ import logger from '@wdio/logger'
 
 const log = logger('@wdio/crossbrowsertesting-service')
 
-export default class CrossBrowserTestingLauncher {
-    constructor (options, caps, config) {
-        this.options = options
-        this.config = config
-    }
+export default class CrossBrowserTestingLauncher implements WebdriverIO.ServiceInstance {
+    private _tunnel: boolean = false;
+    private _cbtTunnelOpts: CBTConfigInterface = undefined as any;
+
+    constructor (
+        private _options: WebdriverIO.ServiceOption,
+        private _caps: WebDriver.DesiredCapabilities,
+        private _config: WebdriverIO.Config
+    ) {}
```

Comment:
better define this in the constructor
```suggestion
    ) {
        this._cbtTunnelOpts = Object.assign({
            username: this._config.user,
            authkey: this._config.key,
            nokill: true
        }, this._options.cbtTunnelOpts)
    }
```